test/ng/filter/orderBySpec.js

```diff
@@ -157,6 +183,38 @@ describe('Filter: orderBy', function() {
     });
 
 
+    it('should support a custom comparator', function() {
```

Comment:
I feel that the test is a bit too clever with "ownerD" being the nerd in question. ;)